content/whats-new-tinacloud/2024.11.41.mdx

```diff
@@ -0,0 +1,8 @@
+---
+versionNumber: 2024.11.41
+dateReleased: 2024-11-26T22:12:50Z
+---
+
+## What's Changed
+
+* Code - ♻️ Refactoring release
```

Comment:
duplicate of the version before

Reply:
Not quite - different release, same aim.

Reply:
✅